### PR TITLE
Update the rust version for rust wasm tutorial

### PR DESCRIPTION
**Pull request by wtlin1228**
### Description

Update the rust version used in the `Compiling from Rust to WebAssembly` page.

### Motivation

Reduce the mismatches when readers follow this tutorial.

### CODE REVIEW

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Hi @wtlin1228, and thank you for updating this! I've run through the tutorial with your changes and tested that it still works.

I have a couple of suggestions that I thought you could make while you are here, for a bit of extra improvement:

- You should change "Firstly, run the following command:" to "First, run the following command inside your `hello-wasm` directory:"
- In the "A digression about code size" section it says "If you check out the generated WebAssembly code size, it may be a few hundred kilobytes.". When I ran through the tutorial to test it, it was only 17KB, so I'm assuming that this has improved massively since the tutorial was first published. Maybe delete this section?

What do you think?

**Comment by wtlin1228** [comment]
Hi @chrisdavidmills, good suggestions! I'll go through the tutorial and update it as much as I can.

File: files/en-us/webassembly/rust_to_wasm/index.md [hunk 1]
```diff
@@ -183,13 +190,13 @@ Now that we've got everything set up, let's build the package.
...
 For this purpose, we'll use the [`--target` argument](https://rustwasm.github.io/docs/wasm-pack/commands/build.html#target) in `wasm-pack build` to specify what kind of WebAssembly and JavaScript is generated.
 
-Firstly, run the following command:
+Firstly, run the following command inside your `hello-wasm` directory:
 
 ```bash
 wasm-pack build --target web
 ```
 
-This does a number of things (and they take a lot of time, especially the first time you run `wasm-pack`). To learn about them in detail, check out [this blog post on Mozilla Hacks](https://hacks.mozilla.org/2018/04/hello-wasm-pack/). In short, `wasm-pack build`:
```
Comment (wtlin1228) [inline files/en-us/webassembly/rust_to_wasm/index.md:192]: The first run took 8s only

**Review by chrisdavidmills — APPROVED** [review]
Nice work @wtlin1228, your changes look great.

I also took it upon myself to copy edit the page a bit, as some of the language was a bit sloppy.

Let's get this merged!

**Pull request merged**